HHS/Head-Start-TTADP
# [TTAHUB-2098] Monitoring data widget backend

## Description of change
Add a backend query for the monitoring data widget

## How to test
Confirm that the data returned from the service is retrieved from the seeds properly and matches the specs of the Jira ticket.

Note that the CI failures are coming from upstream. If you run the added test, it passes.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2098


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
